modules/nw-dns-operator.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+// * networking/dns/dns-operator.adoc
+
+[id="dns-operator-{context}"]
+= DNS Operator
+
+The DNS Operator implements the `dns` API from the `operator.openshift.io` API
+group. The operator deploys CoreDNS using a DaemonSet, creates a Service for
+the DaemonSet, and configures the kubelet to instruct pods to use the CoreDNS
+Service IP for name resolution.
+
+.Procedure
+The DNS Operator is deployed during installation as a Kubernetes `Deployment`.
+Use the `oc get` command to view the Deployment status:
+----
+$ oc get -n openshift-dns-operator deployment/dns-operator
+NAME           READY     UP-TO-DATE   AVAILABLE   AGE
+dns-operator   1/1       1            1           23h
+----
+
+ClusterOperator is the Custom Resource object which holds the current state
+of an operator. This object is used by operators to convey their state to
+the rest of the cluster. Use the `oc get` command to view the state of the
+DNS Operator:
+----
+$ oc get clusteroperator/dns
+NAME      VERSION                         AVAILABLE   PROGRESSING   DEGRADED   SINCE
+dns       4.1.0-0.okd-2019-05-06-152942   True        False         False      92m
```

Comment:
I'm not sure the version format of the release to customer, but `4.1.0-0.okd-2019-05-06-152942` looks weird. Can we use something like `4.1.0-0.11` or `4.1.0-m.n`?